config/crd/bases/infrastructure.cluster.x-k8s.io_nutanixmachines.yaml

```diff
@@ -309,8 +305,9 @@ spec:
                 description: Will be set in case of failure of Machine instance
                 type: string
               nodeRef:
-                description: NodeRef is a reference to the corresponding workload
-                  cluster Node if it exists.
+                description: 'NodeRef is a reference to the corresponding workload
```

Comment:
<pre>make manifests</pre> needs to be run to regenerate crd/bases

Reply:
assuming this was generated using that and not hand modified?

Reply:
Correct. `manifests` is a dependency of `prepare-local-clusterctl`, and I ran the latter in order to confirm the CRD deployment into the cluster.